packages/cosmic-proto/package.json

```diff
@@ -15,19 +15,25 @@
   },
   "type": "module",
   "exports": {
-    "./swingset/msgs.js": "./gen/agoric/swingset/msgs.js",
-    "./swingset/msgs.ts": "./gen/agoric/swingset/msgs.ts",
-    "./swingset/query.js": "./gen/agoric/swingset/query.js",
-    "./swingset/query.ts": "./gen/agoric/swingset/query.ts",
-    "./swingset/swingset.js": "./gen/agoric/swingset/swingset.js",
-    "./swingset/swingset.ts": "./gen/agoric/swingset/swingset.ts",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./swingset/msgs.js": {
+      "types": "./dist/agoric/swingset/msgs.d.ts",
+      "default": "./dist/agoric/swingset/msgs.js"
+    },
+    "./swingset/query.js": {
+      "types": "./dist/agoric/swingset/query.d.ts",
+      "default": "./dist/agoric/swingset/query.js"
+    },
+    "./swingset/swingset.js": {
+      "types": "./dist/agoric/swingset/swingset.d.ts",
+      "default": "./dist/agoric/swingset/swingset.js"
+    }
```

Comment:
Could we reduce duplication by exporting the copy in `swingset` and not include `dist` in the published artifact? Multiple physical locations could effect multiple module instances and an identity discontinuity between them in the same program.

Reply:
Good call. We don't really need `dist` in the source tree. Force-pushed this change and will merge if it passes CI.